kornia/geometry/transform/elastic_transform.py

```diff
@@ -17,6 +18,7 @@ def elastic_transform2d(
     alpha: Tuple[float, float] = (1.0, 1.0),
     align_corners: bool = False,
     mode: str = 'bilinear',
+    padding_mode: str = SamplePadding.ZEROS.name
```

Comment:
```suggestion
    padding_mode: str = 'zeros',
```

Reply:
fixed, I will run test locally to check that everything passes to don't disturb you again